logging_http_client/http_session.py

```diff
@@ -37,12 +37,14 @@ def __init__(self, source: str, logger: Logger) -> None:
             self.options,
         ]
 
+        logging_level = config.get_logging_level()
+
         for method in methods_to_decorate:
             method_name = method.__name__
-            decorated_method = self.decorate(source, logger, method)
+            decorated_method = self.decorate(source, logger, method, logging_level)
```

Comment:
**issue(blocking):** The default logging level for the client is a global config, no need to extend the signature to access the value. Instead, you can access it directly via `self.logging_level`